chapter_explainability/overview.md

```diff
@@ -0,0 +1,64 @@
+
+**Model-agnostic methods vs. model-specific methods:**
+Based on the applicability of explanation methods, we have model agnostic explanation methods and model-specific explanation methods. The former is more flexible and can be applied to any black-box model regardless of its structure, while the latter is limited to specific model classes.
+
+
+## Desired Properties of Explanations
+There are no standard evaluation measures on explainability. Still, the following dimensions, which might be challenging to quantify rigorously, can be considered as guiding intuitions when developing or evaluating explanation methods:
+
+* Fidelity (or Faithfulness): How well does the explanation model capture the black box models. For instance, if we use a surrogate model to mimic a black-box model, we can check if it can give the same outputs as the original model. We can also measure the fidelity by checking whether the prediction changes if the supposedly important features are flipped.
+* Comprehensibility: To what extent the explanations are humanly comprehensible. Explanations should be aligned to the target user's vocabulary. An explanation that is comprehensible to machine learning engineers or domain experts does not necessarily be understandable to common users.
+* Stability: The extent to which the explanations are similar for similar inputs. The explanations of two similar inputs shall not differ from one another drastically.
+* Transparency: It is the opposite of opaque or black boxes. Transparency can be achieved at three levels: simulatability, decomposability, and algorithmic transparency :cite:`Lipton.2018`. A model is simulatable if a human user can comprehend the entire model at once. A model is decomposable if each of its parts (e.g., input, parameters, calculation) admits an intuitive explanation. Finally, algorithmic transparency means that the learning process used to develop a model is understandable.
+
+Other dimensions such as simplicity (simpler explanations are generally preferred), causality (whether it reflects the causal relationships), and generality (whether the method requires restrictions) might also be taken into consideration.
+
+Note that explanations only present a partial view of black-box models. It is not about explaining everything but understanding enough to accomplish the task. If the explanation model can mimic everything, we will not need black models anymore.
```

Comment:
black models?

Reply:
It is not about explaining everything but understanding enough to accomplish the task.
->
The goal is not to explain everything, but to provide explanations just enough for accomplishing a task.